frontend/src/app/AppRoutes.tsx

```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import { Route, Routes } from 'react-router-dom';
 import ApplicationsPage from '~/pages/ApplicationsPage';
 import UnauthorizedError from '~/pages/UnauthorizedError';
+import DependencyMissingPage from '~/pages/dependencies/DependencyMissingPage';
```

Comment:
Need to revert the changes in this file -- we need the lazy import for route chunking.

Reply:
arg missed that. will do